### DISCUSSION TITLE
Setting up ARM with multiple drives with Docker...

### DISCUSSION THREAD

**Discussion by J-Farmer** [automatic-ripping-machine/automatic-ripping-machine#1124] [Q&A]
Hello! I'm no stranger to ARM or Docker in general, but I'm trying  to do something different that may work, it may not. 

I have 3 drives in my system: a 120 GB SSD for the OS, Docker, ARM, etc. Using just this drive I quickly ran into issues with ripping multiple dvds at once, and I was having to stop and copy files roughly every 4 movies. I added two 250 GB NVMe drives to this system as additional drives. 

I cannot figure out how to set ARM to utilize these drives that are mounted at "/media/arm/working" and "/media/arm/completed" as a transcode and finished drive, respectively. If anyone could point me in the right direction, I'd be appreciative.

**Comment by mihawk90** [comment]
I don't know if you ever solved this but I just stumbled on this post while looking for something else.

What you want to do can be easily accomplished by just mounting the two drives onto ARM's `completed` and `raw` directories respectively. That's not something ARM specific, just how Docker (and Linux in general) work.
The wiki has a general guide on what volumes the Docker container needs mounted [in the Docker Guide](https://github.com/automatic-ripping-machine/automatic-ripping-machine/wiki/docker#understanding-docker-volumes-for-arm).

You can simply add 2 more volumes:
```sh
    -v "/media/arm/completed:/home/arm/media/completed" \
    -v "/media/arm/working:/home/arm/media/raw" \
```

Now, since you can only mount a host-path (i.e. `/media/arm/…`) to one container path (i.e. `/home/arm/media/…`) you'll have to adjust the `TRANSCODE_PATH` in ARMs settings to live under one of the two (I'd recommend the `raw` directory) *if* you want the transcode files to go to your additional drives as well. Otherwise it'll just go to the drive that you're using for `/home/arm/media`.

The Docker guide does everything through the Docker CLI (hence the `-v` option), but you can accomplish the same thing in [Docker Compose](https://docs.docker.com/compose/). If you're using some kind of GUI frontend you'll have to figure out how to add volumes on that.
